content/accessibility/checklist.mdx

```diff
@@ -0,0 +1,148 @@
+
+#### Length of text
+Though a level AAA guideline, try to hit the following typography standards for better ease of reading:
+- Width of text should not exceed 80 characters or glyphs 
+- Don't justify text
+- Allow for text magnification up to 200% without requiring horizontal scrolling.
+
+Read Primer's [typography](https://primer.style/design/foundations/typography) guidelines for more detail about using type.
+
+#### Links
+Navigating from a list of all the links on a given web page is very common for assistive technology users. Make sure that the link text itself is meaningful and unique. There should be as few duplicated references as possible to prevent confusion. 
+
+For more detail on accessible links, read the W3C's [Understanding 2.4.4 - Link Purpose](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html).
+
+<DoDontContainer stacked>
+  <Do>
+```html
+<p>Find out more about <a href="#url">GitHub Enterprise pricing</a>.<br /></p>
+```
+    <Caption>
+      
+    </Caption>
+  </Do>
+  <Dont>
+```html
+<p>
+  To find out more about GitHub Enterprise pricing,
+  <a href="#url">click here</a>.
+</p>
+```
+    <Caption>
+      
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Forms
+- All form inputs should have a label.
+- Don't use placeholder text for titles, hints, or other important information. 
+
+### Alt-text
+When screen reader users come across images, graphs, and icons, by default the screen reader will speak the name of the file, which most of the time can be unhelpful (for example: ``final-final-truly-final-V2.jpg``). 
+
+Use HTML the `alt` attribute when labeling an image. If the image has a descriptive caption already, you should give the image an atttibute of `alt=""`. This will prevent the screenreader from telling the user about the image multiple times. 
+
+## Semantic markup
+Always use semantic HTML elements, like `button`, `ul`, `nav`. Most modern browsers implement the accessibility features outlined in the specs for these elements; without them, elements will need additional [ARIA attributes and roles](https://www.w3.org/TR/wai-aria-1.2/) to be recognized by assistive technologies.
+
+Elements like `h1`-`h6`, `nav`, `footer`, `header` have [meaningful roles](https://www.w3.org/TR/wai-aria-1.2/#role_definitions) assigned, so use them, and use them carefully. This can help assistive technologies read the page better and help users find information quicker.
+
+Only use a `div` or a `span` to markup up content when there isn't another HTML element that would semantically be more appropriate, or when an element is needed exclusively for applying CSS styles or JavaScript behaviors.
+
+<DoDontContainer stacked>
```

Comment:
@ashygee @colebemis I added the _dodont_ component here, but it's not doing what I expected 😬 Should I just remove the code formatting and leave it as text?

Reply:
@yaili this might have to do with the version of Sochi at installed. I'll defer to @colebemis to check it out fully though.